views/about.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/style.css">
+    <title>About</title>
+</head>
+<body>
+    <nav>
+        <ul class = "titles">
+            <li> <a class="nav-bar-item-a" href="/views/home.html"> Home </a> </li>
```

Comment:
Here, you shouldn't use `/views/home.html`, instead, you should use just what you passed on app.js, so here, it would be `/home`